test/connectivity/modules/NetworkController/LinuxFirewallOfflineController.cs

```diff
@@ -0,0 +1,122 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace NetworkController
+{
+    using System;
+    using System.Net;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Microsoft.Extensions.Logging;
+
+    class LinuxFirewallOfflineController : IController
```

Comment:
This class is going to change in order to fix the behavior that when disable network, it just cuts connections to/from Internet, but not between modules. right?

Reply:
It works between containers, no change is expected, it adds a filter by iotHub IP

---
In reply to: [359640559](https://github.com/Azure/iotedge/pull/2137#discussion_r359640559) [](ancestors = 359640559)